### DISCUSSION TITLE
Set Display Order for Next Display/Previous Display

### DISCUSSION THREAD

**Discussion by classicmac** [rxhanson/RectanglePro-Community#544] [Q&A]
Is there a way to set the display order for the Next Display and Previous Display key commands?

**Comment by rxhanson** [comment]
Currently there is not. Are you looking to set it because you don't like the order that it is being put in or it is incorrect, or do you have something unusual about your setup? Just trying to figure out if it's something best fixed without having to lean on more configuration.

**Reply by classicmac** [reply]
It is an issue with 3 or more monitors. When I have them lined upside by side I would like windows to cycle through monitors from left to right. I have had instances where it is not cycling through monitors in that order so windows are jumping to monitors in a non-intuitive manner.
